lib/data_migrate/helpers/infer_test_suite_type.rb

```diff
@@ -0,0 +1,15 @@
+module DataMigrate
+  module Helpers
+    class InferTestSuiteType
+      def call
+        if File.exist?(Rails.root.join('spec', 'spec_helper.rb'))
```

Comment:
We could perform add to the conditionals here for `Rails.configuration.generators.options[:rails][:test_framework] == :rspec` and `Rails.configuration.generators.options[:rails][:test_framework] == :test_unit` respectively to look for both installation and configuration.

Reply:
I like the idea of getting the test framework from Rails's config. I've worked on projects where both rspec and minitest were used (migration happening but not over and very little bandwidth to finish it) and I think its easier for us to maintain compatibility with the configuration over the presence of files?

Perhaps adding a config line would work too, but I'd imagine projects running on sinatra for example to maybe not follow the conventions set elsewhere?